daemon/list.go

```diff
@@ -61,6 +64,21 @@ func (daemon *Daemon) Containers(config *ContainersConfig) ([]*types.Container,
 			}
 		}
 	}
+
+	if ancestors, ok := psFilters["ancestor"]; ok {
+		ancestorFilter = true
+		byParents := daemon.Graph().ByParent()
+		// The idea is to walk the graph down the most "efficient" way.
+		for _, ancestor := range ancestors {
+			// First, get the imageId of the ancestor filter (yay)
+			image, err := daemon.Repositories().LookupImage(ancestor)
+			if err == nil {
```

Comment:
From my scan of the code, I think its ok to ignore it.  I think we'd only get an error if there's something wrong with the graph, which shouldn't really prevent this command from running. We may get less results than expected, but that's better than just stopping.  Perhaps logging it (in the LookupImage() ) func would be good though so an admin watching the logs can investigate.